npm-packages/meteor-babel/options.js

```diff
@@ -186,11 +186,8 @@ function getDefaultsForNode8(features) {
       require("@babel/plugin-syntax-object-rest-spread"),
       require("@babel/plugin-proposal-object-rest-spread")
     );
-    // TODO [fibers]: instead of removing the code below, consider this comment:
-      // https://github.com/meteor/meteor/pull/12471/files#r1089610144
-    const ignoreAsyncPlugin = process.env.IGNORE_ASYNC_PLUGIN === '1';
 
-    if (!features.useNativeAsyncAwait && !ignoreAsyncPlugin) {
+    if (features.useNativeAsyncAwait === false) {
```

Comment:
Is this a breaking change? Before, not setting useNativeAsyncAwait (leaving it as undefined) would cause the async-await plugin to be added.

I had updated everywhere meteor-babel is used in Meteor 3 to set useNativeAsyncAwait to true, so it should be fine to keep the old behavior.

Reply:
@zodern I don't think it would be a breaking change (when comparing with all the changes the 3.0 version will have) - it's just that `fibers` aren't the default now, so I don't think it would make much sense to add the plugin here even if the `useNativeAsyncAwait` feature flag is `undefined`.

Yes, the changes you did already skipped the plugin addition, but I think this is better for newer code that we can add in the future that may use `babel` (so that we don't always need to pass the feature flag) - and we also remove an `env var` that is not needed anymore.

Reply:
Actually I think we should remove the `beta` from the versioning, but I think it would be better to do that on the end.

About renaming the option, I think it makes sense, but IMHO it's out of the scope from this PR - here we just want to have it running without `env vars`.